drake_ros_examples/examples/rgbd/BUILD.bazel

```diff
@@ -0,0 +1,75 @@
+# -*- mode: python -*-
+# vi: set ft=python :
+
+load("@ros2//:ament_index.bzl", "ament_index_share_files")
+load("@ros2//:ros_cc.bzl", "ros_cc_binary")
+load("@ros2//:ros_py.bzl", "ros_py_binary")
+load("@ros2//:ros_py.bzl", "ros_py_test")
+
+# We use `ament_index_share_files` so we can write code that can find
+# resources both when built by CMake and by Bazel.
+ament_index_share_files(
+    name = "rgbd_share",
+    package_name = "drake_ros_examples",
+    srcs = [":rgbd.sdf"],
+    strip_prefix = "examples/",
+)
+
+ros_cc_binary(
+    name = "rgbd",
+    srcs = ["rgbd.cc"],
+    data = [
+        ":rgbd_share",
+    ],
+    local_defines = ["BAZEL"],
+    rmw_implementation = "rmw_cyclonedds_cpp",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@drake//geometry:drake_visualizer",
+        "@drake//geometry/render_vtk:factory",
+        "@drake//multibody/parsing:parser",
+        "@drake//multibody/plant:multibody_plant_config_functions",
+        "@drake//systems/analysis:simulator",
+        "@drake//systems/framework:diagram_builder",
+        "@drake//systems/sensors:camera_info",
+        "@drake//systems/sensors:rgbd_sensor",
+        "@drake//visualization:visualization_config",
+        "@drake//visualization:visualization_config_functions",
+        "@drake_ros_repo//:drake_ros",
+        "@drake_ros_repo//core",
+        "@drake_ros_repo//viz",
+        "@gflags",
+        "@ros2//:rclcpp_cc",
+        "@ros2//:sensor_msgs_cc",
+    ],
+)
+
+ros_py_binary(
```

Comment:
I'm not able to run this locally : 
```
$ bazel run //examples/rgbd:rgbd_py
INFO: Analyzed target //examples/rgbd:rgbd_py (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
ERROR: /c224acaf994c8660d7350147cfd32cf5/external/drake/bindings/pydrake/BUILD.bazel:55:37: Action external/drake/bindings/pydrake/documentation_pybind.h failed: (Exit 1): mkdoc failed: error executing command (from target @drake//bindings/pydrake:generate_pybind_documentation_header) bazel-out/k8-opt-exec-2B5CBBC6/bin/external/drake/tools/workspace/pybind11/mkdoc -DDRAKE_COMMON_SYMBOLIC_EXPRESSION_DETAIL_HEADER -DEIGEN_MPL2_ONLY -DSPDLOG_SHARED_LIB -DSPDLOG_COMPILED_LIB ... (remaining 861 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
Traceback (most recent call last):
  File "/c224acaf994c8660d7350147cfd32cf5/sandbox/linux-sandbox/5091/execroot/drake_ros_examples/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/drake/tools/workspace/pybind11/mkdoc.runfiles/drake/tools/workspace/pybind11/mkdoc.py", line 816, in <module>
    main()
  File "c224acaf994c8660d7350147cfd32cf5/sandbox/linux-sandbox/5091/execroot/drake_ros_examples/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/drake/tools/workspace/pybind11/mkdoc.runfiles/drake/tools/workspace/pybind11/mkdoc.py", line 644, in main
    add_library_paths(parameters)
  File "c224acaf994c8660d7350147cfd32cf5/sandbox/linux-sandbox/5091/execroot/drake_ros_examples/bazel-out/k8-opt-exec-2B5CBBC6/bin/external/drake/tools/workspace/pybind11/mkdoc.runfiles/drake/tools/workspace/pybind11/libclang_setup.py", line 43, in add_library_paths
    raise RuntimeError(f'Library file {library_file} does NOT exist')
RuntimeError: Library file /usr/lib/x86_64-linux-gnu/libclang-12.so does NOT exist
Target //examples/rgbd:rgbd_py failed to build
Use --verbose_failures to see the command lines of failed build steps.
```

Reply:
I was able to run the simple ``rgbd`` file, but not this python one.

Reply:
Needed to install libclang-12-dev, but this worked when I tried in a fresh container.